src/config/index.ts

```diff
@@ -10,7 +10,7 @@ import {DevSecurityLevel} from '@shardus/core'
 
 crypto.init('69fa4195670576c0160d660c3be36556ff8d504725be8a59b5a96509e0c994bc')
 
-export const networkAccount = '0'.repeat(64)
+export const networkAccount = '1000000000000000000000000000000000000000000000000000000000000001'
```

Comment:
Ohh I see. I wasn't aware that it would be used in other repos as well.

I was just updating it the `100...001` because it is hardcoded in the archiver.
https://github.com/shardus/archive-server/blob/2fc70b7c4de0d6afa128ecffe20e5af29dede60b/src/Config.ts#L146

But yeah... makes sense to keep using the all zeros address. I will revert this change and will make changes in the archiver repo.

Reply:
Thanks. The global account address being hard-coded in the archiver should be fixed too.

Reply:
reverted it to use the all zeros address. plz check.